### PR TITLE
v1.14.4

### PR DESCRIPTION
**Pull request by fujimotos**
* This is the pull request for upcoming v1.14.4 release.
* If anything is missing from my release note, please comment here.
  (I'm going to fast-forward this branch tomorrow)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,20 @@
+# v1.14.4
+
+## Release v1.14.4 - 2022/01/06
+
+### Enhancement
+
+* in_tail: add option to skip long lines (max_line_size)
+  https://github.com/fluent/fluentd/pull/3565
+
+### Bug fix
```
Comment (ashie) [inline CHANGELOG.md:10]: Please add #3560 too

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,20 @@
...
+
+* out_file: Make append option work without compression
```
Comment (ashie) [inline CHANGELOG.md:12]: This bug reproduces only on macOS, I want to add a note for it.
See also `Releaes Note` in https://github.com/fluent/fluentd/pull/3579#issue-1085787780

**Review by ashie — CHANGES_REQUESTED** [review]
Please

File: CHANGELOG.md (hunk 1 above)
Comment (kenhys) [reply CHANGELOG.md:10]: It is useful if we could explain a bit more. (when it should be used)

**Comment by kenhys** [comment]
It looks good if the above 2 issues were fixed.

**Review by ashie — APPROVED** [review]

**Comment by fujimotos** [comment]
So I was poking around the `CHANGELOG.md` this morning. I ended adding a few fixes:

 - The bug-fix section contains more detailed explanation about what was fixed.
    - e.g. `On macOS with Ruby 2.7/3.0, out_file fails to write events if append is true.` 
 - Use backticks to escape underscore characters.

File: CHANGELOG.md (hunk 1 above)
Comment (fujimotos) [reply CHANGELOG.md:10]: I added the item to the release note in d1f33da.

File: CHANGELOG.md (hunk 2 above)
Comment (fujimotos) [reply CHANGELOG.md]: d1f33da added the affected macOS and Ruby versions.

**Review by kenhys — APPROVED** [review]
LGTM.

**Comment by fujimotos** [comment]
Merged. Thank you!

**Pull request merged**